.buildkite/dagster-buildkite/dagster_buildkite/utils.py

```diff
@@ -112,7 +112,7 @@ def get_python_versions_for_branch(pr_versions=None):
     # landing and on release branches before shipping
     branch_name = os.getenv("BUILDKITE_BRANCH")
     assert branch_name is not None, "$BUILDKITE_BRANCH env var must be set"
-    if branch_name == "master" or is_release_branch(branch_name):
+    if branch_name == "master" or is_release_branch(branch_name) or "fullbuild" in branch_name:
```

Comment:
It's probably easier to use `BUILDKITE_MESSAGE` for this? https://buildkite.com/docs/pipelines/environment-variables#BUILDKITE_MESSAGE? E.g., check if `fullbuild` is in the test plan (a simple `in` check will do)

Otherwise, if you forget to put `fullbuild` in your branch name, you'll have to create another PR? I Don't believe you can change the branch name once you've made a PR.

Reply:
ooo neat didnt know about that

could do both

Reply:
Good point-- wasn't sure whether you can change the branch name of PR, but better to avoid the hassle in any case.

What if we check both `BUILDKITE_MESSAGE` and the branch name? According to linked doc `BUILDKITE_MESSAGE` is "usually the commit message". This is a little unclear. If it's just the most recent commit message then this could get annoying (have to include in every commit message) when you have a branch with a feature that you want to give the full treatment to.

Reply:
yeah both is fine. wish there was a `BUILDKITE_PULL_REQUEST_DESCRIPTION` for this, since you're right that `BUILDKITE_MESSAGE` is for the most recent commit.